lib/scylla_cloud_io_setup.py

```diff
@@ -185,6 +185,36 @@ def generate(self):
             self.disk_properties["read_bandwidth"] = 1656354602 * nr_disks
             self.disk_properties["write_iops"] = 92233 * nr_disks
             self.disk_properties["write_bandwidth"] = 1028010325 * nr_disks
+        elif self.idata.instancetype == "i4i.xlarge":
+            self.disk_properties["read_iops"] = 109695 * nr_disks
+            self.disk_properties["read_bandwidth"] = 804797120 * nr_disks
+            self.disk_properties["write_iops"] = 60301 * nr_disks
+            self.disk_properties["write_bandwidth"] = 559770112 * nr_disks
+        elif self.idata.instancetype == "i4i.2xlarge":
+            self.disk_properties["read_iops"] = 218603 * nr_disks
+            self.disk_properties["read_bandwidth"] = 1607257984 * nr_disks
+            self.disk_properties["write_iops"] = 92206 * nr_disks
+            self.disk_properties["write_bandwidth"] = 1128130304 * nr_disks
+        elif self.idata.instancetype == "i4i.4xlarge":
+            self.disk_properties["read_iops"] = 392047 * nr_disks
+            self.disk_properties["read_bandwidth"] = 3116210176 * nr_disks
+            self.disk_properties["write_iops"] = 92208 * nr_disks
+            self.disk_properties["write_bandwidth"] = 2289037056 * nr_disks
+        elif self.idata.instancetype == "i4i.8xlarge":
+            self.disk_properties["read_iops"] = 696555 * nr_disks
+            self.disk_properties["read_bandwidth"] = 6229529600 * nr_disks
+            self.disk_properties["write_iops"] = 183782 * nr_disks
+            self.disk_properties["write_bandwidth"] = 4580575744 * nr_disks
+        elif self.idata.instancetype == "i4i.16xlarge":
+            self.disk_properties["read_iops"] = 1253572 * nr_disks
+            self.disk_properties["read_bandwidth"] = 8557109248 * nr_disks
```

Comment:
This doesn't seem correct. 16xlarge has twice as many disks as 8xlarge, but it will have four times the bandwidth (you doubled both the constant and nr_disks).